packages/quick_actions/quick_actions_ios/ios/Classes/FLTQuickActionsPlugin.m

```diff
@@ -4,7 +4,7 @@
 
 #import "FLTQuickActionsPlugin.h"
 
-static NSString *const CHANNEL_NAME = @"plugins.flutter.io/quick_actions";
+static NSString *const CHANNEL_NAME = @"plugins.flutter.io/quick_actions_ios";
```

Comment:
As long as you are touching this, could you fix it to `kChannelName`?

Reply:
Replaced with the correct convention.